tardis/plasma/properties/property_collections.py

```diff
@@ -1,5 +1,15 @@
+from tardis.plasma.properties import (BetaRadiation, LevelBoltzmannFactor,
+    Levels, Lines, SelectedAtoms, AtomicMass, LTEPartitionFunction,
+    LevelPopulationLTE, LevelNumberDensity, PhiSahaLTE, GElectron,
+    IonizationData, NumberDensity, IonNumberDensity, LinesLowerLevelIndex,
+    LinesUpperLevelIndex, TauSobolev, TRadiative, AtomicData, Abundance,
+    Density, TimeExplosion)
 
 class LTEInputs(list):
-    pass
-
-
+    def __init__(self):
```

Comment:
I inherit from list so you can just do `self.append`. But other than that it looks good. you sort of see the concept of what I want, right?